slides/webpack.config.js

```diff
@@ -29,9 +29,10 @@ module.exports = {
     new HtmlWebpackPlugin({
       template: './index.html'
     }),
-    new CopyWebpackPlugin([ 
-      { from: { glob: 'content/*.md' } },
-      { from: { glob: 'content/images/*' } }
+    new CopyWebpackPlugin([
+      { from: { glob: 'content/**/*.md' } },
+      { from: { glob: 'content/images/*' } },
+      { from: { glob: 'content/bios/images/*' } }
```

Comment:
OK, this is where I spotted that you've put stuff in new sub-directories - rather than having a second `images` sub-directory, how about putting both `.md` and images in `content/bios`?